lib/evm/eei.ts

```diff
@@ -7,11 +7,17 @@ const { VmError, ERROR } = require('../exceptions')
 
 export default class EEI {
   _env: any
+  _runState: any
+  _result: any
   _state: PStateManager
+  _interpreter: any
```

Comment:
Test comment to see if a comment I do on a certain commit diff page will show up on the overall "Files changed" PR tab.

Reply:
Update: yes, it does.

(Background: thought if it might make sense to do comments on a per-commit-basis. Seems not possible though. Would have eventually eased the review process.)

Reply:
@s1na Ok, will just delete this again, probably been sent by mail, enough for short taking note.